nevow/js/Nevow/Athena/__init__.js

```diff
@@ -105,6 +105,15 @@ Nevow.Athena.PageWidget.methods(
         var d = self.remoteCalls[responseId];
         delete self.remoteCalls[responseId];
 
+        if (d === undefined) {
+            // If the transport is running then something strange has happened,
+            // otherwise just do nothing.
+            if (self.deliveryChannel.running) {
+                Divmod.msg('Received response to missing call: ' + responseId);
```

Comment:
"missing" is a little mysterious if you're not deeply familiar with this bit of the code already; how about "unknown or already-responded call" instead?